features/in_onderzoek.feature

```diff
@@ -696,143 +808,190 @@ Functionaliteit: in onderzoek
+      | inOnderzoek.datumIngangOnderzoekVerblijfplaats.langFormaat | 7 maart 2022  |
 
       #hier betekent leeg dat het inOnderzoek gegeven niet is opgenomen in het antwoord
       Voorbeelden:
-      | gegeven in onderzoek      | waarde | adresregel1 | adresregel2 | korteNaam | straat | woonplaats |
-      | gemeenteVanInschrijving   | 080910 |             | true        |           |        | true       |
-      | straat                    | 081110 | true        |             | true      | true   |            |
-      | naamOpenbareRuimte        | 081115 |             |             |           | true   |            |
-      | huisnummer                | 081120 | true        |             |           |        |            |
-      | woonplaats                | 081170 |             | true        |           |        |            |
-      | locatiebeschrijving       | 081210 | true        |             |           |        |            |
-      | adresregel1               | 081330 | true        |             |           |        |            |
-      | groep gemeente            | 080900 |             | true        |           |        | true       |
-      | groep adres               | 081100 | true        | true        | true      | true   | true       |
-      | groep locatiebeschrijving | 081200 | true        |             |           |        |            |
-      | groep verblijf buitenland | 081300 | true        | true        |           |        |            |
-      | categorie verblijfplaats  | 080000 | true        | true        | true      | true   | true       |
+      | gegeven in onderzoek      | waarde | adresregel1 | adresregel2 | korteNaam | straat | woonplaats | datum      | datum type | datum langFormaat |
+      | gemeenteVanInschrijving   | 080910 |             | true        |           |        | true       | 2022-03-07 | Datum      | 7 maart 2022      |
+      | straat                    | 081110 | true        |             | true      | true   |            | 2022-03-07 | Datum      | 7 maart 2022      |
+      | naamOpenbareRuimte        | 081115 | true        |             |           | true   |            | 2022-03-07 | Datum      | 7 maart 2022      |
+      | huisnummer                | 081120 | true        |             |           |        |            | 2022-03-07 | Datum      | 7 maart 2022      |
+      | woonplaats                | 081170 |             | true        |           |        | true       | 2022-03-07 | Datum      | 7 maart 2022      |
+      | locatiebeschrijving       | 081210 | true        |             |           |        |            |            |            |                   |
+      | regel 1 adres buitenland  | 081330 | true        |             |           |        |            |            |            |                   |
+      | groep gemeente            | 080900 |             | true        |           |        | true       | 2022-03-07 | Datum      | 7 maart 2022      |
```

Comment:
woonplaats wordt niet gevuld uit gemeente, dus moet die hier niet in onderzoek staan

Reply:
aangepast

Reply:
woonplaats moet niet true (niet in onderzoek) zijn